processing/src/main/java/io/druid/query/aggregation/Aggregator.java

```diff
@@ -41,7 +41,11 @@
   Object get();
   float getFloat();
   long getLong();
-  double getDouble();
+
+  default double getDouble()
+  {
+    return (double) getFloat();
```

Comment:
can we add a comment like `this default method is add to enable smooth backward compatibility, please re-implement it if you aggregators work with numeric double columns` or something similar.